src/main/java/decodes/sql/ConfigListIO.java

```diff
@@ -812,16 +815,21 @@ private void ingestRow(ResultSet rs, PlatformConfig pc)
 			if (s != null && s.length() > 0)
 				dataOrder = s.charAt(0);
 		}
-
-		DecodesScript ds;
-
-		ds = new DecodesScript(pc, name);
-		ds.setDataOrder(dataOrder);
-		ds.setId(id);
-		readFormatStatements(ds);
```

Comment:
`readFormatStatements` is no longer used. Method can get deleted

Reply:
done.